CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 2.29.0-rc.0 / 2021-07-29
+
+* [CHANGE] Remove experimental tag on `--storage.tsdb.allow-overlapping-blocks` flag. #9117
+* [CHANGE] Remove experimental tag on `--storage.tsdb.retention.size` flag. #8728
+* [FEATURE] Add Kuma service discovery. #8844
+* [FEATURE] Add `__meta_gce_interface_ipv4_<name>` meta label to GCE discovery. #8978
```

Comment:
For new labels in SDs and other minor new features, I also had them as `FEATURE` initially for the last release, but @roidelapluie's opinion was that they should be treated as an `ENHANCEMENT` instead, see discussion in https://github.com/prometheus/prometheus/pull/8954#issuecomment-863285982.

At least for consistency, we would want to downgrade them to `ENHANCEMENT` here too, but I'm also not sure if we should keep that policy.